templates/kubernetes.html

```diff
@@ -0,0 +1,570 @@
+          <li class="p-list__item is-ticked">モニタリングとログ記録</li>
+          <li class="p-list__item is-ticked">ライフサイクル管理</li>
+          <li class="p-list__item is-ticked">バックアップとリカバリ</li>
+        </ul>
+        <p><a href="/contact-us?product=kubernetes-explorer">Kubernetes Explorerに関するお問い合わせ&nbsp;&rsaquo;</a></p>
+      </div>
+      <div class="col-6 p-card">
+        <div class="p-card__header">
+          <h2 class="p-heading--four">Kubernetes Discoverer</h2>
+          <p><span class="p-heading--two">45,000ドル</span><br />1回限りの料金</p>
+        </div>
+        <p>仮想化環境、プライベートクラウド、パブリッククラウドでのデプロイなど、お客様の要件に合わせたカスタムアーキテクチャを 1 週間で構築して運用します。</p>
+        <p>内容:</p>
+        <ul class="p-list">
+          <li class="p-list__item is-ticked">可用性の高いKubernetesをパブリッククラウド、VMware、OpenStackでデプロイ</li>
+          <li class="p-list__item is-ticked">お客様のワークロードに合わせて最適化されたカスタムKubernetesアーキテクチャ</li>
+          <li class="p-list__item is-ticked">Calico、Canal、Flannelのネットワーキング</li>
+          <li class="p-list__item is-ticked">3 日間の対面でのCanonical Kubernetesオンサイトトレーニング</li>
+        </ul>
+        <p><a href="/contact-us?product=kubernetes-discoverer">Kubernetes Discovererについてのお問い合わせ先&nbsp;&rsaquo;</a></p>
+      </div>
+      <div class="col-6 p-card">
+        <div class="p-card__header">
+          <h2 class="p-heading--four">Kubernetes Discoverer Plus</h2>
+          <p><span class="p-heading--two">95,000ドル</span><br />1回限りの料金</p>
+        </div>
+        <p>お客様の要件に適したモジュール式のエコシステムによる本番稼働レベルのKubernetesクラスターを、3週間で構築して運用します。仮想化環境またベアメタルをデプロイします。</p>
+        <p>内容:</p>
+        <ul class="p-list">
+          <li class="p-list__item is-ticked">可用性の高い本番稼働レベルのKubernetesをパブリッククラウド、VMware、OpenStack、ベアメタルでデプロイ</li>
+          <li class="p-list__item is-ticked">GPUアクセラレーション</li>
+          <li class="p-list__item is-ticked">永続ボリューム用ストレージ</li>
+          <li class="p-list__item is-ticked">カスタムネットワーキングのオプション</li>
+          <li class="p-list__item is-ticked">管理プラットフォーム</li>
+          <li class="p-list__item is-ticked">プライベートレジストリ</li>
+          <li class="p-list__item is-ticked">ロードバランサー</li>
+          <li class="p-list__item is-ticked">アプリケーションカタログ</li>
+          <li class="p-list__item is-ticked">2日間の対面でのKubernetesオペレーターへのオンサイトでの知識提供</li>
+        </ul>
+        <p><a href="/contact-us?product=kubernetes-discoverer-plus">Kubernetes Discoverer Plusに関するお問い合わせ&nbsp;&rsaquo;</a></p>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-strip--light is-bordered">
+    <div class="row">
+      <div class="col-12">
+        <h3>DiscovererおよびDiscoverer Plus向けのAI（人工知能）/ ML（機械学習）アドオン</h3>
+        <p>機械学習のツールと運用を幅広く理解するためにK8sワークショップを活用してください。開発者ワークステーションからデータセンターやパブリッククラウドまで、完全なAI分析パイプラインの構築が可能です。GPGPUの統合、パブリッククラウドの最適化、KubeflowやTensorflowなどのツールが対象です。ワークショップには独立系のデータサイエンティストによる評価が含まれています。Canonicalはハードウェア、クラウド、データ分析の大手企業との協力により、お客様のAI戦略への統合のための幅広い選択肢を提供しています。</p>
+        <p><a href="/ai">KubernetesとAI/MLに関する詳細&nbsp;&rsaquo;</a></p>
```

Comment:
This link was wrong in the copydoc, I've updated it.
```suggestion
        <p><a href="/ai-ml">KubernetesとAI/MLに関する詳細&nbsp;&rsaquo;</a></p>
```